### ISSUE TITLE
Support scope-down statements in manager rules

### ISSUE THREAD

**Issue by jgournet** [cloudposse/terraform-aws-waf#99]
### Describe the Feature

Currently, scope_down_statement is only supported in rate_based_statement.
It would be great if it was supported in other rules too (I am particularly interested in adding it into managed_rule_group_statement_rules)

### Expected Behavior

Being able to have a scope_down_statement in managed_rule_group_statement_rules

### Use Case

Trying to use this module for managed_rule_group_statement_rules - not having scope_down_statement means I would have to write my own rules, which defeats the purpose of using this module.

### Describe Ideal Solution

I guess it is "just" a big copy/paste job of "scope_down_statement" from rate_based_statement into managed_rule_group_statement_rules.

**Comment by dejwsz** [comment]
yep, it is a must have option there as now the module is limited
